module/userProjects/helper.js

```diff
@@ -372,8 +372,14 @@ module.exports = class UserProjectsHelper {
                     }
                 }
 
-                await kafkaProducersHelper.pushProjectToKafka(projectUpdated);
-
+                // push to kafka only if project is submitted and certificate key is present
+                if ( projectUpdated.status == CONSTANTS.common.SUBMITTED_STATUS &&
+                     projectUpdated.certificate &&
+                     Object.keys(projectUpdated.certificate).length > 0
+                ) {
```

Comment:
@VISHNUDAS-tunerlabs Why are we changing current behavior, project should be pushed to Kafka as before after any sync